### DISCUSSION TITLE
ScriptReference/UIElements.InspectorElement

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#589] [Notes]
# ScriptReference/UIElements.InspectorElement

https://docs.unity3d.com/ScriptReference/UIElements.InspectorElement.html

**Comment by vertxxyz** [comment]
If you're solely drawing an Object using InspectorElement and you don't need any surrounding UI, consider just using [EditorUtility.OpenPropertyEditor](https://docs.unity3d.com/ScriptReference/EditorUtility.OpenPropertyEditor.html) instead.
